docs/design/sno-board.md

```diff
@@ -0,0 +1,50 @@
+# SNOBoard
+
+## OVERVIEW
+
+The SNOBoard is a GUI for Storage Node Operators to get more information about their storage nodes performance as it pertains to the network or individual satellites.
+
+Wireframes:
+https://www.figma.com/file/IlY5UNF94rEpCLGR6BeaOhXX/Local-Storage-Node-Dashboard-Low-Fidelity-Mock?node-id=7%3A520 
+
+## GOALS
+
+- Give SNOs a pretty graphical user interface they can look at to get metrics on:
+	- Node(s) performance per satellite
+	- Storage space
+  	- Bandwidth
+	- Audit information
+	- Uptime information
+  	- Current Status
+	- SN version currently running
+	- DQ’ed status
+	- Containment status
+- Satellite notifications
+	- Notifications center on the SNOBoard
+	- Email notification opt-in
+- Expected payout per satellite, per node
+- Give SNOs a GUI to configure their SNs
+	- Ability to adjust allocated egress/ storage space in the GUI
+	- Ability to graceful exit a satellite or the network in the GUI
+
+## Business Requirements/ Job Stories
+
+- When a SNO starts the CLI dashboard I want the URL for the SNOBoard to be printed on the screen so that the user can easily access the SNOBoard.
+- When my SN is connected to multiple satellites I want the ability to select a satellite on the SNOBoard so that I can drill down into information about my node as it pertains to that satellite.
+	- All satellites should be the default option
+		- This option should NOT show information about uptime or audit checks because those are specific to satellites
+		- The “All Satellites” option should only show information about allocated and used storage space/ bandwidth 
+- When I want to know how much of the allocated storage space my SN has used on the network I want a simple graphic on the SNOBoard so that I can consume this information quickly
+- When I need to know how much of the allocated bandwidth my SN has used I want a simple graphic on the SNOBoard so that I can consume this information quickly
+- When I am looking at the bandwidth graphic I want to be able to distinguish how much bandwidth has been egress vs ingress so that I can determine if my internet plan is suitable for the type of traffic my node is getting. 
+- When a SNO is running an outdated version of SN software I want that information to be displayed on the SNOBoard so that they SNO can determine if they need to upgrade their software
```

Comment:
Who tells the storage node that the version is outdated? Satellite and version control are not designed to do that. They show a minimum version but they don't show the latest version.

Taking the latest version from Github doesn't work because we are not pusing all versions to docker. Github will show a new satellite version that storage nodes don't need to install. Are they outdated just because there is a new satellite version available?

Reply:
See https://github.com/storj/storj/pull/2789